setup/install_gatling/30_classification/90_clamps.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+# Clamps is a module based stress testing tool. It will generate a number of
+# users on a system to simulate agents. It will then randomize their puppet
+# agent -t times to simulate a large number of agents.
+def add_clamps_group_ca(classifier, pe_infra)
+  # The Clamps::master class manages auth.conf for the agents - by setting it
+  # to allow all.  This is insecure and worth looking at overriding at some
+  # point, but this should only be used for metrics / load testing on a private
+  # network, so shouldn't be an issue short term.
+  clamps_ca_group = {
+    'name'    => 'Clamps CA',
+    'rule'    => ['or', ['=', 'name', master.node_name]], # pinned node
+    'parent'  => pe_infra['id'],
+    'classes' => {
+      'clamps::master' => {}
+    }
+  }
+  classifier.find_or_create_node_group_model(clamps_ca_group)
+end
+
+# Creates a node group for creating the fake agents. This node group should
+# match all users that are running as root.
```

Comment:
should that be "only users that are running as root".  saying all implies there are more than one.  And the rule limits it to the metrics node, so there should only be one root user.